actions.js

```diff
@@ -10,7 +10,8 @@ var twitterAPI = require('node-twitter-api'),
     Q = require('q'),
     _ = require('sequelize').Utils._,
     util = require('./util'),
-    setup = require('./setup');
+    setup = require('./setup'),
+    verifyCredentials = require('./verify-credentials');
```

Comment:
Good catch. I missed the invocations in that file even though I added the import :laughing: Fixed in a73a00e31385fbf3382c29d809b66ba92c823f25

I did a project-wide grep and didn't find any other cases where the buUser verifyCredentials function was being invoked.